homeassistant/components/homewizard/manifest.json

```diff
@@ -4,11 +4,9 @@
   "documentation": "https://www.home-assistant.io/integrations/homewizard",
   "codeowners": ["@DCSBL"],
   "dependencies": [],
-  "requirements": [
-    "aiohwenergy==0.8.0"
-  ],
+  "requirements": ["python_homewizard-energy==1.0.0"],
   "zeroconf": ["_hwenergy._tcp.local."],
   "config_flow": true,
   "iot_class": "local_polling",
-  "loggers": ["aiohwenergy"]
+  "loggers": ["homewizard_energy"]
```

Comment:
I am not sure about this one, should it be called by the package name or the class name?

Reply:
The package logs under `homewizard_energy`, but the integration logs under `homewizard`. So probably both?